### PR TITLE
Initial One Click Service

### PR DESCRIPTION
**Pull request by AbdullahAli**
Generating the list of newsletters to the drop down box. Adding a success message with csv download button (with uuid and payload link) then handling the subscription link generated to the csv, linking success and error pages when link is clicked. Removed tests ;)

### CODE REVIEW

File: app/views/pages/index.ejs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<html>
+	<head>
+		<link rel="stylesheet" href="https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-buttons,o-grid,o-errors,o-colors,o-header,o-message,o-forms@5.8.2:/demos/src/demo.scss,o-fonts@^3" />
```
Comment (keirog) [inline app/views/pages/index.ejs]: ~`:/demos/src/demo.scss`~

File: app/views/pages/error.ejs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<html>
+	<head>
+		<link rel="stylesheet" href="https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-buttons,o-grid,o-errors,o-colors,o-header,o-message,o-forms@5.8.2:/demos/src/demo.scss,o-fonts@^3" />
```
Comment (keirog) [inline app/views/pages/error.ejs]: ~`:/demos/src/demo.scss`~

File: app/views/css/style.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+body {
+	background: #fff1e5
```
Comment (keirog) [inline app/views/css/style.css]: +`;`

File: app/controllers/one-click-controller.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+const { validationResult } = require('express-validator/check');
+const UserListService = require('../services/users-list-service');
```
Comment (keirog) [inline app/controllers/one-click-controller.js]: UserListService looks like it’s a class that’s going to be instantiated further down. Any chance we could start controller names with lower case? (Same for some other controllers)

**Comment by AbdullahAli** [comment]
Feedback addressed: https://github.com/Financial-Times/email-one-click/pull/3/commits/227ce272110073d600153e086a22de25e0dd29d9 👍

**Pull request merged**